docs/features/sessions/how-to-ssh-into-a-session.md

```diff
@@ -48,6 +48,14 @@ grid session ssh happy-owl-123
 
 ![](/images/sessions/how-to-ssh-into-session.gif)
 
+:::note
+If trying to use session with ssh and asked for a password: 
+- make sure to do the above Step 0: Create an SSH Key and Step 1: Add the SSH key to Grid
```

Comment:
Hyperlink to step - above. I.e,
"make sure to do the above [Step 0: Create an SSH Key](https://docs.grid.ai/features/sessions/how-to-ssh-into-a-session#step-0-create-an-ssh-key) and [Step 1: Add the SSH key to Grid](https://docs.grid.ai/features/sessions/how-to-ssh-into-a-session#step-1-add-the-ssh-key)"

Reply:
Thanks added the links